client/components/domains/register-domain-step/index.jsx

```diff
@@ -1163,7 +1161,6 @@ class RegisterDomainStep extends React.Component {
 						onButtonClick={ this.onAddDomain }
 						pendingCheckSuggestion={ this.state.pendingCheckSuggestion }
 						unavailableDomains={ this.state.unavailableDomains }
-						isEligibleVariantForDomainTest={ this.props.isEligibleVariantForDomainTest }
 						selectedFreePlanInSwapFlow={ this.props.selectedFreePlanInSwapFlow }
 						selectedPaidPlanInSwapFlow={ this.props.selectedPaidPlanInSwapFlow }
 						isReskinned={ this.props.isReskinned }
```

Comment:
For a site on a paid plan, when I enter the launch flow, the initial suggestions and search results look like this:

<img width="1080" alt="Screenshot 2020-09-23 at 12 47 46 PM" src="https://user-images.githubusercontent.com/1269602/93980856-cd575680-fd9c-11ea-88e3-9b858c220c25.png">

<img width="1238" alt="Screenshot 2020-09-23 at 1 00 56 PM" src="https://user-images.githubusercontent.com/1269602/93980890-d9431880-fd9c-11ea-8db7-25fb0fbfd272.png">

The domain price should not be listed there, instead we should see the "Registration free" text here since the user is already on a paid plan. I noticed that this bug exists on production too, so you don't need to fix it in this PR. If you prefer to tackle it as a separate PR, could you please track the issue in a new GH issue? Thanks.

Reply:
Tracked @ 639-gh-martech